octobot_evaluators/matrix/matrix_manager.py

```diff
@@ -150,6 +160,23 @@ def get_tentacles_value_nodes(matrix_id, tentacle_nodes, cryptocurrency=None, sy
             if node_at_path is not None]
 
 
+def get_latest_eval_time(matrix_id, exchange_name=None, tentacle_type=None, cryptocurrency=None,
```

Comment:
Where is it used?

Reply:
In scripted trading modes to get the timestamp of the latest evaluation. This timestamp is later used to select values in cache